tests/jest/pageScanner/context.test.js

```
@@ -0,0 +1,62 @@
+/**
+ * Test for scanner context exclusions
+ * 
+ * This test verifies that elements with the selectors in the exclude array
+ * are properly excluded from accessibility scans, including the newly added
+ * #qm-icon-container selector.
+ */
```

ESLint

[error] Expected JSDoc block lines to be aligned.

(jsdoc/check-line-alignment)